apps/fulfillment/project.json

```diff
@@ -20,7 +20,7 @@
       "options": {
         "commands": [
           "rm -rf dev-dist/sw",
-          "npx rollup --config rollup.sw.config.mjs"
```

Comment:
Can we remove `rollup` dependency from the monorepo and from the FA `package.json` now?